libs/directus/directus-next-component/src/components/DirectusVideo/index.tsx

```
@@ -1,18 +1,17 @@
+  const fileProps = getDirectusFile(rest, baseUrl)
 
-  const fileProps = getDirectusFile(props, baseUrl)
-
-  if (fileProps == null)
+  if (isNullish(fileProps))
     return null
 
   const { type } = fileProps
 
-  return <Video {...fileProps} type={type ?? undefined} />
+  return <Video {...omit(fileProps, ['filenameDownload', 'duration', 'caption', 'embed', 'filenameDownload', 'filename_disk', 'filename_download', 'focal_point_divider', 'focal_point_x', 'focal_point_y', 'location', 'storage', 'tags', 'thumbhash'])} type={type ?? undefined} />
```

ESLint

[error] Unsafe return of a value of type error.

(ts/no-unsafe-return)